src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -47,7 +47,9 @@
     private static final File CEUTRIO_20_21_GATK3_4_G_VCF = new File(largeFileTestDir, "gvcfs/CEUTrio.20.21.gatk3.4.g.vcf");
     private static final String CEUTRIO_20_21_EXPECTED_VCF = "CEUTrio.20.21.gatk3.7_30_ga4f720357.expected.vcf";
     private static final List<String> ATTRIBUTES_TO_IGNORE = Arrays.asList(
+            "AN",
             "AS_QD",
+            "RGQ",
```

Comment:
Why do we need these new exclusions?

Reply:
Good question. GATK4 doesn't appear to use `AN`, nor have code that [moves GQ to RGQ](https://github.com/broadinstitute/gsa-unstable/blob/c707a5d4d55da9add817793b98b4d1e089151414/protected/gatk-tools-protected/src/main/java/org/broadinstitute/gatk/tools/walkers/variantutils/GenotypeGVCFs.java#L483) like GATK3 did. So these are removed from the comparison with GATK3 output. @ldguathier - is is correct that GATK4 doesn't do this ?

Reply:
I'm not sure anyone actually used RGQ.  We probably didn't port that feature.

I don't know why we're ignoring AN though.  That should be part of the ChromosomeCounts annotation (also AC and AF) that's part of the standard group.  What happens if you don't ignore AN?

Reply:
@ldgauthier GATK4 isn't adding `AN` because `ChromosomeCounts` isn't `Reducible` (so its [bypassed by the annotation engine here](https://github.com/broadinstitute/gatk/blob/67f0f0f2e59185b721398b17c24eba487a2ac76c/src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java#L217)). Perhaps thats a separate problem ?

Reply:
But in your testSpanningDeletion output, there's an AC=2 and AN=2 on the deletion, which must have come from ChromosomeCounts.  Is it just not being called consistently?  We don't need it at non-variant sites.

Reply:
I updated the output files and removed the `AN` test exclusion.